web/src/pm_list_data.ts

```diff
@@ -178,12 +185,17 @@ export function get_list_info(
         // the unzoomed view.
         return false;
     }
-    for (const [idx, conversation] of conversations.entries()) {
-        if (should_show_conversation(idx, conversation)) {
+
+    for (const conversation of conversations) {
+        if (should_show_conversation(conversation)) {
             conversations_to_be_shown.push(conversation);
         } else {
             more_conversations_unread_count += conversation.unread;
         }
+
+        if (!conversation.is_deactivated) {
+            visible_conversations_count += 1;
```

Comment:
Why does this happen even when `should_show_conversation` is false? The conversation isn't visible, after all.